pages/about-html-plugin

```diff
@@ -35,6 +35,31 @@
       "type": "html",
       "id": "a15eb154dbd1c13d",
       "text": "Use &lt;h3>...&lt;/h3> for headings. By convention we use size 3."
+    },
+    {
+      "type": "html",
+      "id": "dc5098757cfc4c3e",
+      "text": "<h3> Forms </h3>"
+    },
+    {
+      "type": "paragraph",
+      "id": "8a1c7ced09e2415c",
+      "text": "Use <form ...> ... <input type=submit ...></form> for posts to remote services. Services are expected to respond with json for a page to be added to the lineup."
+    },
+    {
+      "type": "paragraph",
+      "id": "555a2d4d5f96b0b9",
+      "text": "For example, this form submits an image url to a remote service as if the image had been dropped on an image transporter. See [[About Transport Plugin]]"
+    },
+    {
+      "type": "html",
+      "id": "f80adaee5751ad1e",
+      "text": "<form \n  method=post\n  action=\"http://fed.wiki.org:4010/image\"\n  style=\"background-color:#eee; padding:15px;\">\n<center>\nTransport Image to Wiki Page<br><br>\nurl:\n<input\n  name=text\n  size=40\n  value=\"http://c2.com/wiki/wiki-21st.jpg\">\n<input\n  name=html\n  type=hidden\n  value=\"\">\n<input\n  type=submit\n  value=submit>\n</center>\n</form>"
+    },
+    {
+      "type": "paragraph",
+      "id": "4319e863cb1ceb36",
+      "text": "Note: fidelity with the Transport plugin api is not required of form handlers."
     }
```

Comment:
New about page section is as follows:

![image](https://cloud.githubusercontent.com/assets/12127/14584189/c69689b2-03ee-11e6-9c0b-fd42d435deff.png)